internal/servers/server/server_test.go

```diff
@@ -544,18 +554,22 @@ func TestShutdown(t *testing.T) {
 	}
 
 	tests := []test{
-		{
-			name: "returns nil when server is not running",
-			checkFunc: func(s *server, got, want error) error {
-				if want != got {
-					return errors.Errorf("Shutdown returns error: %v", got)
-				}
-				return nil
-			},
-			want: nil,
-		},
-
 		func() test {
+			return test{
+				name: "returns nil when server is not running",
+				checkFunc: func(s *server, got, want error) error {
```

Comment:
Do you have any idea for check ing`s *server`?
In this case, how about checking the value of `s.running`?

Reply:
Yes, It looks good!
I will fix the code based on your suggestion!

Reply:
fixed: https://github.com/vdaas/vald/pull/1689/commits/966482aea81afa1bee8e304407d77265c683ab28